python/aida_interchange/aifutils.py

```diff
@@ -371,6 +371,13 @@ def make_hypothesis(g, hypothesis_uri, hypothesis_content, system):
     return hypothesis
 
 
+def mark_importance(g, entity, importance):
```

Comment:
If this is the case, I recommend we make this change in the Java version as well, since I used that method as a reference. src/main/java/com/ncc/aif/AIFUtils.java line 953

Reply:
Agreed that could be renamed as well. The java version concerns me less because you can see that entity is a Resource object whereas in python you have no idea.